index.js

```diff
@@ -0,0 +1,103 @@
+const titleInput = document.querySelector('.titleInput');
```

Comment:
## Section Specific Requirement.
- [ ] Do you mind keep the application logic in this module separate from DOM manipulation tasks just as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright? 

**We call this `Separation of Concerns`. An idea that each module or layer in an application should only be responsible for one thing and should not contain code that deals with other things. Separating concerns reduces code complexity by breaking a large application down into many smaller units of encapsulated functionality.**

Reply:
I do understand the concern you are trying to explain I completed the project as it was simple and short but if its needed I will change and if I can continue as I have done then it will be great at this point. will try to follow the concern from the next milestone.